static/app/views/dashboardsV2/widgetBuilder/header.tsx

```diff
@@ -63,9 +63,9 @@ export function Header({
         <ButtonBar gap={1}>
           <Button
             title={t(
-              "You’re seeing the metrics project because you have the feature flag 'organizations:metrics' enabled. Send us feedback via email."
+              "You are seeing the widget builder on a full page because you have the feature flags 'organizations:new-widget-builder-experience' and 'dashboards-edit' enabled. How do you like the new widget builder? Send us feedback via email."
```

Comment:
```suggestion
              "How do you like the new widget builder? Send us feedback via email."
```
Do we need to go into detail about feature flags?